config/routes.rb

```diff
@@ -1,3 +1,13 @@
 Rails.application.routes.draw do
+  root "apartments#index"
+  get ":about" => "misc#about", as: :about
```

Comment:
Does this route need a dynamic path segment? It doesn't appear that you're expecting one in the controller.